deploy/template.yaml

```diff
@@ -51,10 +51,11 @@ Conditions:
   IsBuild: !Equals [ !Ref AWS::AccountId, "457601271792"]
   IsNotDevelopment: !Not [ !Condition IsDevelopment ]
   IsProduction: !Equals [ !Ref Environment, production ]
-  IsSubscriptionEnviroment: !Or
+  IsSubscriptionEnvironment: !Or
```

Comment:
I think this was named because staging and above are the Splunk subscription envs, but given this is used in multiple use cases it would be clearer if it was just named what it actually is i.e. `IsStagingIntProd` ?